### PR TITLE
Add information about apps listed in deploy API

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/apidocs-mxsdk/apidocs/deploy-api-4.md [hunk 1]
```diff
@@ -110,6 +110,18 @@ The following steps will change the permissions of a team member to an environme
...
+### 4.1 Unexpected Apps Listed by API
+
+This API call does not return the same results as you can see within the Developer Portal. It includes all licensed and free apps on the Mendix Cloud. Free apps include all apps which have been created or edited in Studio Pro, even if they have then been deployed to a different cloud such as Mendix for Private Cloud.
+
+In the Developer Portal:
+
+* The [Nodes](/developerportal/deploy/node-permissions/#nodes) screen in the Developer Portal shows all the licensed apps which are returned by this request, but does not show any Free Apps.
+
+* The [My Apps](/developerportal/#my-apps) screen shows both licensed apps and Free Apps, but also includes apps which are deployed to other platforms (for example, Mendix for Private Cloud or SAP BTP) and Free Apps which have not yet been deployed and therefore have no environments set up for them.
+
```
Comment (MarkvanMents) [inline content/en/docs/apidocs-mxsdk/apidocs/deploy-api-4.md]: Would this be better in the description of `GET /apps` rather than the supporting document?

Comment (Luyao-Zhang-1) [reply content/en/docs/apidocs-mxsdk/apidocs/deploy-api-4.md]: I think that's a good idea. Have you also discussed with the developer?

Comment (MarkvanMents) [reply content/en/docs/apidocs-mxsdk/apidocs/deploy-api-4.md]: It is waiting to be reviewed. Hopefully @mbyregowMx will respond when he reviews this.

Comment (mbyregowMx) [reply content/en/docs/apidocs-mxsdk/apidocs/deploy-api-4.md]: We could move this to the `Get /apps` description itself for v4 API

Comment (MarkvanMents) [reply content/en/docs/apidocs-mxsdk/apidocs/deploy-api-4.md]: @mbyregowMx 
Could you update the v4 API yaml with this information and send it to me (over Slack?) and I can then update the documentation that way instead.
Thanks.

Comment (MarkvanMents) [reply content/en/docs/apidocs-mxsdk/apidocs/deploy-api-4.md]: New yaml received and text is now in the Swagger, so this section is removed.

**Comment by MarkvanMents** [comment]
Publication agreed over Slack.

**Pull request merged**